### ISSUE TITLE
Gregicality multiblocks using old metal casing

### ISSUE THREAD

**Issue by ivelieu** [GregTechCEu/gregicality#617]
I'm on Gregicality skyblock edition version 6.1, which is using Gregicality-1.12.2-0.23.1. I can't find any previous mention of this issue, and to be honest I am not sure if the issue belongs here or on https://github.com/GregTechCEu/gregicality-multiblocks/pulls (which I found from a link in the description of the gregicality bugs channel on the GTCEu discord) 

The casings for the large miner and advanced large miner are using the old casing types with no recipe. They should be using the "Sturdy HSS-G" and "Rigid HSS-S" casings. It seems that they were missed for edits or not existing at the time of #507, maybe this is where it was missed: https://github.com/GregTechCEu/gregicality/pull/507#pullrequestreview-717326252

The "Basic Large Miner" does have the correct "Rugged black steel casing" though. 

![image](https://user-images.githubusercontent.com/28797826/147901125-48db4ecd-227c-4d38-b159-d0e13783275f.png)
![image](https://user-images.githubusercontent.com/28797826/147901156-f7981ab7-7a3f-4ef8-842b-cd348e582081.png)

**Comment by TechLord22** [comment]
I believe the latest version of that modpack, `Gregicality Skyblock Edition 6.3`, has an updated version using `Gregicality-1.12.2-0.23.3`. In that version, the casings used are correct. Let us know if updating does not fix this.

**Comment by ivelieu** [comment]
Github GSE newest is 6.1. I'm 90% sure whoever uploaded to curseforge did different versioning. Which I can say has caused me a lot of confusion >_< 

Curseforge 6.3 is older than github 6.1.

https://github.com/GregTechCEu/gregtech-skyblock-edition/releases
![image](https://user-images.githubusercontent.com/28797826/148484361-4f8459bb-fc7f-40e3-882f-f4e1f80e067f.png)

https://www.curseforge.com/minecraft/modpacks/gregicality-skyblock-edition/files
![image](https://user-images.githubusercontent.com/28797826/148484499-b6acf150-a988-4e48-af04-5f9bb92d837c.png)

Here I make a fresh download of GSE 6.3 via curseforge in multiMC and we can see it uses the same version Gregicality-1.12.2-0.23.1
![image](https://user-images.githubusercontent.com/28797826/148484736-d4ae63f8-a9fa-4f79-a7da-a8477f918924.png)


I will also mention I *did* check Gregicality 1.12.2-0.23.2 and 1.12.2-0.23.3 patch notes before making this post. They do not list anything about the casings getting fixed. 
![image](https://user-images.githubusercontent.com/28797826/148484890-dcaa5e98-2050-434a-bfc4-703e8d7bc7cf.png)
![image](https://user-images.githubusercontent.com/28797826/148484933-a617bce0-f87e-48a7-96e7-0e357dc00c5e.png)

Just in case what you were saying is right aside all of that, I have made a new instance of the github GSE 6.1 and replaced the gregicality jar to 1.12.2-0.23.3. Indeed the issue still persists with the updated version.
![image](https://user-images.githubusercontent.com/28797826/148485696-6bac5b87-3f94-4a03-a694-5aa6d7ff0297.png)
![image](https://user-images.githubusercontent.com/28797826/148485699-2938c5e0-a498-49cd-8495-3399c4204016.png)

**Comment by TechLord22** [comment]
Okay honestly this is pretty funny. I'll see if updating Gregicality locally will fix that (and not cause problems), and if so, will push an update to the modpack accordingly.

**Comment by TechLord22** [comment]
I have released a new version of Gregicality Skyblock Edition on github: <https://github.com/GregTechCEu/gregtech-skyblock-edition/releases/tag/6.4>

This updates Gregicality to the latest release, and additionally fixes the issue with the miners.

**Comment by ivelieu** [comment]
Thank youuuu ❤️  all fixed~

**Comment by ivelieu** [comment]
So, I didn't think of it at the time, but advancing further I realised some other multiblocks have the same issue. I looked through all the multiblocks in gregicality to check. There are three others: the large extruder, large packaging machine and large wiremill. 

I was hoping to fix the issue myself after seeing how @TechLord22 did it, I spent some time looking at gregicality\src\main\java\gregicadditions\GAConfig.java and gregtech-skyblock-edition/config/gtadditions.cfg and was feeling very puzzled. Although I was not able to work it out, I did notice some typos in the config comments.

https://github.com/GregTechCEu/gregicality/blob/fc7068ee70a7973f379b326aa7578c366f9f7fd2/src/main/java/gregicadditions/GAConfig.java#L772
https://github.com/GregTechCEu/gregicality/blob/fc7068ee70a7973f379b326aa7578c366f9f7fd2/src/main/java/gregicadditions/GAConfig.java#L862
https://github.com/GregTechCEu/gregicality/blob/fc7068ee70a7973f379b326aa7578c366f9f7fd2/src/main/java/gregicadditions/GAConfig.java#L893

![image](https://user-images.githubusercontent.com/28797826/149646068-d4dd257f-1078-4527-86f5-8a5aeb1e2bdf.png)
![image](https://user-images.githubusercontent.com/28797826/149646069-414fa8cc-3fe1-495b-87a3-d637e6461c23.png)
![image](https://user-images.githubusercontent.com/28797826/149646070-a041d071-fd05-4985-a6b5-ef2c187b7093.png)

**Comment by ivelieu** [comment]
I found the fix ✨ 

I also make a pull request here to update the documentation generator for gtadditions.cfg. I don't know how to compile the gtadditions.cfg file from the GAConfig.java annotations but I tried to edit both to make them line up. 

I do kind of wonder why HG-1223 casing is prefixed "toxic", I mean, it's only used for a packager, I'm not sure I want all my little dusts rubbing onto toxic casings. 😝
